number_parser/parser.py

```diff
@@ -158,6 +158,38 @@ def _normalize_dict(lang_dict):
     return {_strip_accents(word): number for word, number in lang_dict.items()}
 
 
+def _is_cardinal_token(token, lang_dict):
+    if token in lang_dict.all_numbers:
+        return token
+    return None
+
+
+def _is_ordinal_token(token, lang_dict):
+    token = _apply_cardinal_conversion(token, lang_dict)
+    return _is_cardinal_token(token, lang_dict)
+
+
+def _is_number_token(token, lang_dict):
+    return _is_cardinal_token(token, lang_dict) or _is_ordinal_token(token, lang_dict)
```

Comment:
I prefer to put the `language` as an argument instead of the `lang_dict` to make it easier to use it.

For `_is_number_token()` I think we can do it public: `is_number_token()`, as my intention is to use it when integrating this library with `dateparser`.